hot/log.js

```diff
@@ -3,11 +3,7 @@ var logLevel = "info";
 function dummy() {}
 
 function shouldLog(level) {
-	var shouldLog =
-		(logLevel === "info" && level === "info") ||
-		(["info", "warning"].indexOf(logLevel) >= 0 && level === "warning") ||
-		(["info", "warning", "error"].indexOf(logLevel) >= 0 && level === "error");
-	return shouldLog;
+	return typeof level === "string" && ["info", "warning", "error"].includes(level);
```

Comment:
includes can't be used, old browsers do not support it

Reply:
oh i forgot it 🤨🙄